deploy/helm/cht-chart/templates/api-deployment.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    cht.service: api
+  name: cht-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      cht.service: api
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 0
+      maxUnavailable: 1
+  template:
+    metadata:
+      labels:
+        cht.service: api
+    spec:
+      containers:
+        - env:
+            - name: BUILDS_URL
+              value: https://staging.dev.medicmobile.org/_couch/builds_4
```

Comment:
MIght it be worth externalizing this to `Values.yml`, particularly if production builds pull their build config from a different URL?

Reply:
I'd been moving back and forth on this. i.e. considered suggesting retiring staging.dev and using a more modern solution. However it would take a non-trivial amount of work. As it stands, that value can't be configurable because it's pretty much hard coded in the builds. @dianabarsan might have more to say here.